packages/devextreme/js/ui/tree_view/ui.tree_view.base.js

```diff
@@ -1432,6 +1442,22 @@ const TreeViewBase = HierarchicalCollectionWidget.inherit({
         });
     },
 
+    _removeClickEvent: function(itemsContainer) {
+        const { eventName, clickSelector, pointerDownEvent, pointerDownSelector } = this._getEventsData();
+
+        eventsEngine.off(itemsContainer, eventName, clickSelector);
+        eventsEngine.off(itemsContainer, pointerDownEvent, pointerDownSelector);
+    },
+
+    _getEventsData: function() {
```

Comment:
[naming]
The name seems misleading for me - we don't get data for all events here.
```suggestion
    _getItemClickEventData: function() {
```